py-polars/src/interop/numpy/to_numpy_series.rs

```diff
@@ -276,6 +275,12 @@ fn series_to_numpy_with_copy(py: Python, s: &Series, writable: bool) -> PyResult
         },
         List(_) => list_series_to_numpy(py, s, writable),
         Array(_, _) => array_series_to_numpy(py, s, writable),
+        Struct(_) => {
+            let ca = s.struct_().unwrap();
+            let df = ca.clone().unnest();
+            // TODO: How should we determine the IndexOrder here?
+            df_to_numpy(py, &df, IndexOrder::Fortran, writable, true).unwrap()
```

Comment:
For struct it should always be `Fortran` I think. That fits our memory representation best.